client.go

```diff
@@ -118,3 +124,30 @@ func newClientUsingAddresses(port uint16, remoteAddresses []string) (*Client, er
 		return newClientUsingCluster(cluster)
 	}
 }
+
+// Get is a simple invocation of the NewFetchValueCommand
+func (c *Client) Get(opts *GetOptions) ([]*Object, error) {
```

Comment:
To be consistent this should be named `Fetch` rather than `Get`. I'm not sure if an entirely separate options struct is necessary as there are only three options that are also the same as will be used by `Delete`.